cpp/ql/src/Likely Bugs/Protocols/TlsSettingsMisconfiguration.qhelp

```diff
@@ -6,6 +6,17 @@
   <p>Using TLS or SSLv23 protool from the boost::asio library, but not disabling deprecated protocols or disabling minimum-recommended protocols.</p>
 </overview>
 
+<recommendation>
+<p>When using the TLS or SSLv23 protocol, set the <code>SSL_OP_NO_TLSv1</code> and <code>SSL_OP_NO_TLSv1_1</code> options, but do not set <code>SSL_OP_NO_TLSv1_2</code>. When using the SSLv23 protocol, also set the <code>SSL_OP_NO_SSLv3</code> option.</p>
```

Comment:
Why are the options here (`SSL_OP_NO_TLSv1` etc.) in upper case when they are lower-cased and fully qualified in the examples (`boost::asio::ssl::context::no_tlsv1` etc.)?

Reply:
Good point - I've changed them to be a bit more consistent.  I'm not familiar enough with the library to be sure how they're normally referred to.